scripts/build.sh

```diff
@@ -41,18 +41,23 @@ rm -rf ./proto/.git
 # for a local SimNet network
 if [ "$ARG" == "local" ]; then
   echo "Copying env simnet local file to .env"
-  cp ./.env-simnet-local-sample ./.env
+  cp ./.env-simnet-sample ./.env
   echo "Copying dev file to 'docker-compose.override.yml'"
-  cp ./docker-compose.dev.yml ./docker-compose.override.yml
-# If the build is not local AND the override exists for whatever reason,
-# we should delete the file to not mess w/ the build
+  cp ./docker-compose.simnet.yml ./docker-compose.override.yml
+elif [ "$ARG" == "regtest" ] || [ "$ARG" == "testnet" ] || [ $ARG == "mainnet" ]; then
+  echo "Copying env $ARG local file to .env"
+  cp ./.env-$ARG-sample ./.env
```

Comment:
I don't like copying sample files like this

Reply:
@treygriffith I agree for the broker. We will still have `npm run build local` or `npm run build` and provide the sample files for users during setup. I will remove this line